src/inmanta/ast/statements/define.py

```diff
@@ -134,7 +134,6 @@ def __init__(
             raise HyphenException(lname)
 
         self.anchors = [TypeReferenceAnchor(namespace, x) for x in parents]
```

Comment:
@sanderr this one I don't find how to move into the evaluate function without breaking existing tests. Except for this I think I did all the asked changes.

Reply:
What's the issue? This one doesn't depend on normalization of anything else so we could leave it here if we need to but I'd prefer to move it if we can.

Reply:
Whenever I move it to the evaluate function I have extra anchors in the anchormap. I don't where they come from but the source location is always __internal__:1:1.

Reply:
See 6 lines down

Reply:
So can I leave it there right? I don't think I can move this code block to the evaluate function